File: jscript/g00.array.js
// create empty list
var nicknames = new Array()
// resize array and insert element by index
nicknames[0] = "bob"
nicknames[1] = "ed"
nicknames[2] = "steve"
nicknames[3] = "ralph"
nicknames[4] = "joe"
nicknames[5] = "deb"
nicknames[6] = "kate"

WScript.echo("The total nicknames are: " + nicknames.length);
WScript.stdout.write("The nicknames are: ");
for (var index in nicknames) { 
  if (index == nicknames.length - 1) 
    WScript.Echo(nicknames[index]);
  else 
    WScript.stdout.write(nicknames[index] + ", ");
}

File: lessons/compiled_lang/cs/src/g00.array.cs
using System;

class G00Array
{
    static void Main()
    {
        // populate array one item at a time
        string[] nicknames = new string[7];
        nicknames[0] = "bob";
        nicknames[1] = "ed";
        nicknames[2] = "steve";
        nicknames[3] = "ralph";
        nicknames[4] = "joe";
        nicknames[5] = "deb";
        nicknames[6] = "kate";

        Console.WriteLine("The total nicknames are: " + nicknames.Length);
        Console.WriteLine("The nicknames are: " + string.Join(", ", nicknames));
    }
}
